app/support/auto_logout.rb

```diff
@@ -0,0 +1,38 @@
+class AutoLogout
```

Comment:
I agree. There's also some shared code with the auto cancel job. That was more than I wanted to take on with this ticket. I think a good time to clean this class up further will be when we address why it's not actually toggling the instrument off and the other tickets Aaron mentioned in the ticket.